entity-framework/core/providers/cosmos/functions.md

```diff
@@ -44,23 +42,11 @@ Math.Sin(a)                | SIN(@a)
 Math.Sqrt(d)               | SQRT(@d)
 Math.Tan(a)                | TAN(@a)
 Math.Truncate(d)           | TRUNC(@d)
-MathF.Acos(x)              | ACOS(@x)
-MathF.Asin(x)              | ASIN(@x)
-MathF.Atan(x)              | ATAN(@x)
-MathF.Atan2(y, x)          | ATN2(@y, @x)
-MathF.Ceiling(x)           | CEILING(@x)
-MathF.Cos(x)               | COS(@x)
-MathF.Exp(x)               | EXP(@x)
-MathF.Floor(x)             | FLOOR(@x)
-MathF.Log(x, y)            | LOG(@x, @y)
-MathF.Log(x)               | LOG(@x)
-MathF.Log10(x)             | LOG10(@x)
-MathF.Pow(x, y)            | POWER(@x, @y)
-MathF.Round(x)             | ROUND(@x)
-MathF.Sin(x)               | SIN(@x)
-MathF.Sqrt(x)              | SQRT(@x)
-MathF.Tan(x)               | TAN(@x)
-MathF.Truncate(x)          | TRUNC(@x)
+
+> [!TIP]
+> In addition to the methods listed here, corresponding [generic math](/dotnet/standard/generics/math) implementations
```

Comment:
:+1:

Can put links to the MathF class docs and to the different functions if you want.